src/main/proxy.ts

```diff
@@ -191,11 +190,9 @@ export const launchProxyAndAttachEmitter = async (
   k6StudioState.proxyEmitter.emit('status:change', 'starting')
 
   return launchProxy(browserWindow, k6StudioState.appSettings.proxy, {
-    onReady: async () => {
-      const isProxyHealthy = await checkProxyHealth()
-      const proxyStatus = isProxyHealthy ? 'online' : 'unhealthy'
+    onReady: () => {
       k6StudioState.wasProxyStoppedByClient = false
-      k6StudioState.proxyEmitter.emit('status:change', proxyStatus)
+      k6StudioState.proxyEmitter.emit('status:change', 'online')
```

Comment:
I reverted this back to `online` as the default state. The health check polling will be responsible for changing it if a connection can't be established.